adb_graphics/figures/maps.py

```diff
@@ -406,48 +408,48 @@ def _title(self):
             not_labeled.extend([h.short_name for h in self.hatch_fields])
             if not any(list(set(cf.short_name).intersection(['pres']))):
                 title = cf.vspec.get('title', cf.field.long_name)
-                contoured.append(f'{title} ({cf.units}, hatched)')
+                contoured.append(f'{title} ({cf.units}, hatched)\n')
 
         # Add descriptor string for the important contoured fields
         if self.contour_fields:
             for cf in self.contour_fields:
                 if cf.short_name not in not_labeled:
                     title = cf.vspec.get('title', cf.field.long_name)
+                    title = title.replace("Geopotential", "Geop.")
                     contoured.append(f'{title}')
                     contoured_units.append(f'{cf.units}')
 
-        contoured = ', '.join(contoured)
+        contoured = ''.join(contoured)  # The join method makes contoured a string, enabling linefeeds
         if contoured_units:
-            contoured = f"{contoured} ({', '.join(contoured_units)}; contoured)"
+            contoured = f"{contoured} ({', '.join(contoured_units)}, contoured)"
 
-        # Analysis time (top) and forecast hour (bottom) on the left
-
-        plt.title(f"{self.model_name}: {atime}\nFcst Hr: {f.fhr}",
+        # Analysis time (top) and forecast hour with Valid Time (bottom) on the left
+        plt.title(f"{self.model_name}: {atime}\nFcst Hr: {f.fhr}, Valid Time {vtime}",
                   alpha=None,
                   fontsize=14,
                   loc='left',
                   )
 
-        # Atmospheric level and unit in the high center
         level, lev_unit = f.numeric_level(index_match=False)
-        if not f.vspec.get('title'):
-            level = level if not isinstance(level, list) else level[0]
-            plt.title(f"{level} {lev_unit}", position=(0.5, 1.04), fontsize=18)
-
-        # Two lines for shaded data (top), and contoured data (bottom)
-        title = f.vspec.get('title', f.field.long_name)
         if f.vspec.get('print_units', True):
             units = f'({f.units}, shaded)'
         else:
             units = f''
-        plt.title(f"{title} {units}\n {contoured}",
+
+        # Title or Atmospheric level and unit in the high center
+        if f.vspec.get('title'):
+            title = f.vspec.get('title')
+            plt.title(f"{title} {units}", position=(0.5, 1.06), fontsize=18)
+        else:
+            level = level if not isinstance(level, list) else level[0]
+            plt.title(f"{level} {lev_unit} {f.field.long_name} {units}", position=(0.5, 1.06), fontsize=18)
```

Comment:
To reduce duplication, can you rearrange this so that we only have a single `plt.title(.....` line? If we wanted to move that tile line after this change, we'd need to edit positions in two places, for example.